spec/controllers/new_admin/allocation_chart_controller_spec.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe NewAdmin::AllocationChartController do
+  let(:user) { create(:user) }
+
+  before do
+    allow(controller).to receive(:authenticate_user!)
+    allow(controller).to receive(:current_user).and_return(user)
+  end
+
+  describe 'GET #index' do
+    context 'when there are no allocations' do
+      it 'returns successful status' do
+        get :index
+
+        expect(response).to have_http_status(:ok)
+      end
+
+      it 'returns users within an empty allocation' do
+        get :index
+
+        allocations = subject.instance_variable_get(:@allocations).select { |allocation| allocation.user_id == user.id }
+        user_id = allocations[0].user_id
+
+        expect(user_id).to eq(user.id)
```

Comment:
instead of doing a `select` first and then checking if it's equal, you can just use a `include` matcher: https://www.rubydoc.info/gems/rspec-expectations/RSpec%2FMatchers:include